locale/en.yml

```diff
@@ -781,9 +781,11 @@ en:
       ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript: Job Template (Ansible Tower)
       ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationWorkflow: Workflow Template (Ansible Tower)
       ManageIQ::Providers::AnsibleTower::AutomationManager::Playbook: Playbook (Ansible Tower)
+      ManageIQ::Providers::CloudAutomationManager::ConfigurationManager:    Configuration Manager (Cloud Automation Manager)
```

Comment:
These are pluggable now that we have #20100 merged, so the provider plugins should bring their own dictionary entries.  cc @agrare I think we should blast out these changes to all providers.